packages/gestalt/src/Table.js

```diff
@@ -13,15 +13,17 @@ import TableSortableHeaderCell from './TableSortableHeaderCell.js';
 type Props = {|
   children: React.Node,
   borderSize?: 'sm' | 'none',
+  height?: number | string,
 |};
 
 export default function Table(props: Props) {
-  const { borderSize, children } = props;
+  const { borderSize, children, height } = props;
 
   return (
     <Box
       overflow="auto"
       {...(borderSize === 'sm' ? { borderSize: 'sm', rounding: 1 } : {})}
+      height={height}
```

Comment:
@PayalModi should we change this to `maxHeight` instead? Otherwise it might look off when we set a height higher than the number of items

# Height
![image](https://user-images.githubusercontent.com/127199/84179475-dbc27200-aa3a-11ea-8c34-e90ef43c6038.png)

# Max Height
![image](https://user-images.githubusercontent.com/127199/84179537-f98fd700-aa3a-11ea-816e-0465aaa18360.png)

Reply:
Good call, I updated to have maxHeight instead of height